### PR TITLE
Initial pull request for Satu Mare

### LINKED ISSUE #138: Create sanyee04 hometown
## Satu Mare, RO
- [x] Create a branch
- [x] Create a file
- [x] Create a Pull Request
- [x] Have a conversation
- [x] Get a :+1:
- [x] :ship: it (a.k.a. merge)

### PR DESCRIPTION
**Pull request by peter-sandor-1992**
This PR resolves #138

### CODE REVIEW

File: SatuMare.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Satu Mare is an awesome city.
+It has lots of green areas.
```
Comment (johnj581) [inline SatuMare.txt]: @sanyee04 Good work!

File: SatuMare.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Satu Mare is an awesome city.
```
Comment (Amol-Wankhede) [inline SatuMare.txt:1]: Where it is located?

File: SatuMare.txt (hunk 1 above)
Comment (gerryderry) [inline SatuMare.txt]: sounds great

File: SatuMare.txt [hunk 3]
```diff
@@ -1 +1,2 @@
 Satu Mare is an awesome city.
+It has lots of green areas.
```
Comment (stefanc139) [inline SatuMare.txt]: and scary stories!

File: SatuMare.txt (hunk 1 above)
Comment (cms-jakes) [inline SatuMare.txt]: @sanyee04 I 👍 it!

File: SatuMare.txt (hunk 2 above)
Comment (peter-sandor-1992) [inline SatuMare.txt:1]: @Amol-Wankhede Romania

File: SatuMare.txt (hunk 3 above)
Comment (peter-sandor-1992) [inline SatuMare.txt]: @stefan-chiriac true dat

**Comment by peter-sandor-1992** [comment]
this looks :+1:

**Pull request merged**